### ISSUE TITLE
Parse error: syntax error

### ISSUE THREAD

**Issue by JaoNoctus** [artesaos/defender#102]
**PHP:** v7.0.10
**Laravel:** v5.3.6

source:
```blade
@shield('test')
    ...
@endshield
```

error:
```
Parse error: syntax error, unexpected ''test'' (T_CONSTANT_ENCAPSED_STRING) (View: /home/jaonoctus/htdocs/laravel/resources/views/home.blade.php)
```

log:
```
in 34a69314a1628d6b7321a2436770cbd95c4e44bf.php line 11
at CompilerEngine->handleViewException(object(FatalThrowableError), '1') in PhpEngine.php line 46
at PhpEngine->evaluatePath('/home/jaonoctus/htdocs/laravel/storage/framework/views/34a69314a1628d6b7321a2436770cbd95c4e44bf.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag))) in CompilerEngine.php line 59
at CompilerEngine->get('/home/jaonoctus/htdocs/laravel/resources/views/home.blade.php', array('__env' => object(Factory), 'app' => object(Application), 'errors' => object(ViewErrorBag))) in View.php line 149
at View->getContents() in View.php line 120
at View->renderContents() in View.php line 85
at View->render() in Response.php line 45
at Response->setContent(object(View)) in Response.php line 201
at Response->__construct(object(View)) in Router.php line 1042
at Router->prepareResponse(object(Request), object(View)) in Router.php line 642
at Router->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in SubstituteBindings.php line 41
at SubstituteBindings->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Authenticate.php line 43
at Authenticate->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in VerifyCsrfToken.php line 64
at VerifyCsrfToken->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in ShareErrorsFromSession.php line 49
at ShareErrorsFromSession->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in StartSession.php line 64
at StartSession->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in AddQueuedCookiesToResponse.php line 37
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in EncryptCookies.php line 59
at EncryptCookies->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 104
at Pipeline->then(object(Closure)) in Router.php line 644
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 618
at Router->dispatchToRoute(object(Request)) in Router.php line 596
at Router->dispatch(object(Request)) in Kernel.php line 267
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in Pipeline.php line 137
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 33
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 104
at Pipeline->then(object(Closure)) in Kernel.php line 149
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 116
at Kernel->handle(object(Request)) in index.php line 53
at require_once('/home/jaonoctus/htdocs/laravel/public/index.php') in server.php line 21
```

**Comment by mauri870** [comment]
Thank's for reporting the problem , we will try to reproduce and perform corrections if necessary

**Comment by mauri870** [comment]
I've tested here with a clean installation and it's confirmed, is a bug

**Comment by mauri870** [comment]
The error apparently is originated after the compilation

```
@shield('user.index')
     Hello
@endshield
```
Generate:
```
<?php if(app('defender')->canDo'user.index'): ?>
            Hello
<?php endif; ?>
```

**Comment by mauri870** [comment]
Well, seems like not a bug, but a change implemented only in 5.3

https://github.com/laravel/framework/pull/14986

**Comment by mauri870** [comment]
The bug is fixed on https://github.com/artesaos/defender/commit/511bcf426703808c4daa087e602b5d3b0dbbc54f, I need make some tests after release a new version

Thanks for your report
